projects/storefrontlib/src/lib/product/components/product-list/container/product-list.component.scss

```diff
@@ -1,71 +1,54 @@
 @import 'theme';
-@import 'functions';
-@import 'mixins';
 
-$cx-search-container-trbl-margin: 40px 0 !default;
+.cx-product-search-list {
+  display: block;
+  border-width: var(--cx-border-width, 0 0 1px 0);
+  border-style: var(--cx-border-style, solid);
+  border-color: var(--cx-border-color, var(--cx-g-color-light));
+  margin: var(--cx-padding, 0 0 25px 0);
 
-$cx-search-sorting-border-color: 'light' !default;
-$cx-search-sorting-border: 1px solid !default;
-
-$cx-search-sorting-top-trbl-padding: 0 0 8px 0 !default;
-
-$cx-search-sorting-bottom-trbl-padding: 25px 0 0px 0 !default;
-
-$cx-product-search-dropdown-trbl-padding: 0 35px 0 0 !default;
-$cx-product-search-dropdown-height: 48px !default;
-
-$cx-search-list-border: 1px solid;
-$cx-search-list-border-color: 'light' !default;
-$cx-search-list-trbl-margin: 0 0 25px 0;
-
-.cx-product-search {
-  &-list {
-    display: block;
-    border-bottom: $cx-search-list-border;
-    @include var-color('border-color', $cx-search-list-border-color);
-    margin: $cx-search-list-trbl-margin;
-
-    &:last-of-type {
-      border-bottom: none;
-    }
+  &:last-of-type {
+    border: var(--cx-border, none);
   }
+}
 
-  &__item--container {
-    margin: $cx-search-container-trbl-margin;
-  }
+.cx-product-container {
+  margin: var(--cx-margin, 40px 0);
+}
 
-  &__sorting {
-    &--top {
-      border-bottom: $cx-search-sorting-border;
-      @include var-color('border-color', $cx-search-sorting-border-color);
-      padding: $cx-search-sorting-top-trbl-padding;
+.cx-sorting {
+  &-top {
+    border-width: var(--cx-border-width, 0 0 1px 0);
+    border-style: var(--cx-border-style, solid);
+    border-color: var(--cx-border-color, var(--cx-g-color-light));
+    padding: var(--cx-padding, 0 0 8px 0);
 
-      @include media-breakpoint-down(md) {
-        border-bottom: none;
-      }
+    @include media-breakpoint-down(md) {
+      border: var(--cx-border, none);
     }
+  }
 
-    &--bottom {
-      border-top: $cx-search-sorting-border;
-      @include var-color('border-color', $cx-search-sorting-border-color);
-      padding: $cx-search-sorting-bottom-trbl-padding;
+  &-bottom {
+    border-width: var(--cx-border-width, 1px 0 0 0);
+    border-style: var(--cx-border-style, solid);
+    border-color: var(--cx-border-color, var(--cx-g-color-light));
+    padding: var(--cx-padding, 25px 0 0px 0);
 
-      @include media-breakpoint-down(md) {
-        border-top: none;
-      }
+    @include media-breakpoint-down(md) {
+      border: var(--cx-border, none);
     }
   }
 }
 
-.cx-product-search__pagination {
-  display: inline-block;
-  vertical-align: top;
+.cx-pagination {
```

Comment:
Could you check if this is actually needed. I tried removing it and I didn't see any difference.